src/main/scala/mirroring/config/Config.scala

```diff
@@ -145,6 +147,11 @@ case class Config(
     s"Parameter `primary_key` should be specified if `isChangeTrackingEnabled` is true."
   )
 
+  require(
+    CTChangesQuery.nonEmpty ^ parent_key.length == 0,
```

Comment:
Shouldn't this predicate use && ? As I can understand from message below both conditions must be true...

Reply:
Nope, it's fine, it requires whether `True, True` or `False, False`.

Reply:
Require message has been made more clear.